idunn/api/places_list.py

```diff
@@ -182,16 +207,20 @@ def get_places_bbox(bbox, es: Elasticsearch, indices: IndexNames, settings: Sett
     }
 
 
-def get_events_bbox(bbox, query_params: http.QueryParams):
+def get_events_bbox(bbox: str, category: str, size: int = Query(None)):
```

Comment:
`category` query param should not be required. This may explain why the kuzzle test is failing.

Reply:
Yep it was that. Good catch once again!